packages/self.id-connect/src/services/connectService.ts

```diff
@@ -0,0 +1,296 @@
+/* eslint-disable 
```

Comment:
yeah, only interface needed here should be implementing handlers for the ui provider, then instantiating the 3id-connect service with the provider 

maybe some some confusion here, since not built on the pr that added support for the ui provider in the connect service

cant tell if these files were just moved or also changed

Reply:
I think that this was a personal confusion, I thought that the connect files were needed in order to function, I'm more than happy to kill them. They had some minor changes to them to fit in but again I think that it was just a bit of confusion on my part.